### ISSUE TITLE
Regarding the loss function

### ISSUE THREAD

**Issue by cdb342** [wenjiaXu/APN-ZSL#16]
![2022-06-29 13-40-20 的屏幕截图](https://user-images.githubusercontent.com/61510717/176360128-d49ed603-0cf1-4953-bd07-2fa7a8e4ad06.png)
![2022-06-29 13-40-32 的屏幕截图](https://user-images.githubusercontent.com/61510717/176360151-93d7300e-1536-4e60-8163-fd30d9895a9f.png)
![2022-06-29 13-40-54 的屏幕截图](https://user-images.githubusercontent.com/61510717/176360164-862e2ef7-5f0d-461e-a763-db68464d6450.png)
Hi, it seems there are two softmax calculations in this loss. I am confused about it. Can you tell me the reason for this?
